--- mafft-7.187-with-extensions/extensions/mxscarna_src/Makefile	2013-06-12 02:24:08.000000000 -0700
+++ patch-files/Makefile.mxscarna_src	2014-10-29 10:30:38.603777260 -0700
@@ -6,7 +6,7 @@
 # 1) Choose C++ compiler.
 ################################################################################
 
-CXX = g++
+CXX = icpc
 
 ################################################################################
 # 2) Set C++ flags.
@@ -21,7 +21,8 @@
 
 # no -Wall option for warning
 #CXXFLAGS = -O3 -funroll-loops -finline-limit=$(BIG_INLINE_LIMIT)
-CXXFLAGS = -O3  # by katoh
+#CXXFLAGS = -O3  # by katoh
+CFLAGS = -O3 -no-prec-div -ipo -static AVS_FLAG
 OFLAGS = -DNDEBUG -DNumInsertStates=1 -DVERSION="2.0" $(CXXFLAGS) $(CFLAGS1)
 LIBS = -L$(PROBCONS)  -L./
 INCL = -I$(PROBCONS) -I$(VIENNA) -I./
